src/redux/books/Books.js

```diff
@@ -9,19 +11,62 @@ const reducer = (state = initialState, action) => {
       return [...state, action.payload];
     case REMOVE_BOOK:
       return state.filter((book) => book.id !== action.id);
+    case GET_BOOK: {
+      const books = [];
+      Object.entries(action.payload).forEach(
+        ([key, value]) => books.push({
+          id: key,
+          title: value[0].title,
+          author: value[0].author,
+          genre: value[0].category,
+        }),
+      );
+      return books;
+    }
     default:
       return state;
   }
 };
 
-export const removeBook = (id) => ({
-  type: REMOVE_BOOK,
-  id,
-});
+// export const removeBook = (id) => ({
+//   type: REMOVE_BOOK,
+//   id,
+// });
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+// export const addBook = (payload) => ({
+//   type: ADD_BOOK,
+//   payload,
+// });
```

Comment:
Please ensure that all commented codes are removed from your codebase. It makes your code clean.